### ISSUE TITLE
xsd generated from TEILex0-ODD with UPA errors

### ISSUE THREAD

**Issue by kisram** [DARIAH-ERIC/lexicalresources#223]
We had a similar issue to #66. Creating an xsd using the TEI ODD to XML Schema transformation scenario on oXygen (v 26.0,  build 23100905) results in an xsd file with 23 "Unique Particle Attribution" errors.
Below is the complete list for reference. 
Since we're using ODD chaining, we fixed these on our side, as you can see [here](https://github.com/acdh-oeaw/generic-dict-schema/blob/f9c581cc50246f8987f469990d4e1c19e5859ed0/schema/acdh-ch-dicts.odd#L135). 
It seems that not that many people still use xsd, but since this might be useful for someone else... I tried to identify the reasons for the errors in [this issue](https://github.com/acdh-oeaw/shawi-data/issues/54).

> "http://www.tei-c.org/ns/1.0":xr and "http://www.tei-c.org/ns/1.0":xr (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":xr and "http://www.tei-c.org/ns/1.0":xr (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":note and "http://www.tei-c.org/ns/1.0":note (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":lbl and "http://www.tei-c.org/ns/1.0":lbl (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":lbl and "http://www.tei-c.org/ns/1.0":lbl (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":hi and "http://www.tei-c.org/ns/1.0":hi (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
> "http://www.tei-c.org/ns/1.0":availability and "http://www.tei-c.org/ns/1.0":availability (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.

**Comment by dasch124** [comment]
@kisram we have discussed this in the Lex-0 group. It seems no one else had this problem. When you have time, could you please re-generate the XSD and see whether it is still popping up with the current Lex-0 ODD? Thank you!

**Comment by kisram** [comment]
> [@kisram](https://github.com/kisram) we have discussed this in the Lex-0 group. It seems no one else had this problem. When you have time, could you please re-generate the XSD and see whether it is still popping up with the current Lex-0 ODD? Thank you!
Yes, I still get the same errors with the current ODD (and the relevant parts don't seem to have changed since)
